feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/scenarios/relaychain/StakingRelayChainScenarioInteractor.kt

```diff
@@ -264,13 +262,8 @@ class StakingRelayChainScenarioInteractor(
         require(candidate != null) {
```

Comment:
remove candiadate check - always null in relay chain

Reply:
fixed